ner/src/main/java/edu/illinois/cs/cogcomp/ner/NerTagger.java

```diff
@@ -38,13 +38,13 @@ public static void main(String[] args) {
             Parameters.readConfigAndLoadExternalData(args[args.length - 1], areWeTraining);
 
             if (args[0].equalsIgnoreCase("-train"))
-                LearningCurveMultiDataset.getLearningCurve(-1, args[1], args[2]);
+                LearningCurveMultiDataset.getLearningCurve(-1, args[1], args[2], false);
             else if (args[0].equalsIgnoreCase("-trainFixedIterations"))
                 LearningCurveMultiDataset.getLearningCurve(Integer.parseInt(args[1]), args[2],
-                        args[3]);
+                        args[3], false);
             else {
```

Comment:
is it always the case that if not 'train' or 'trainFixedIterations', we definitely want incremental training?

Reply:
no, but in the "else" clause it is set to 'true' by default -- right?  That is the case I am asking about.

Reply:
There are no calls to getLearningCurve in the else clause, perhaps you are thinking of the "train" argument to ModelLoader.load, which is something else altogether. Although, I may want to change that, so if the model can not be found, it will throw. I will make that change and check it in.

Reply:
I have checked that change in.

Reply:
You're right -- thanks for pointing that out.